src/core/cgn.tf

```
@@ -122,3 +122,57 @@ module "cgn_cosmos_db" {
   account_name        = module.cosmos_cgn.name
 }
 
+
+## Blob storage due to legal backup
+#tfsec:ignore:azure-storage-default-action-deny
+// TODO: verify to use underscore or middlescore
+module "cgn-legalbackup-storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s_legalbackup_storage", local.project), "-", "")
```

```suggestion
  name                       = replace(format("%s_legalbackup_cgn_storage", local.project), "-", "")
```
if this storage it's only for cgn, add cgn suffix in storage name